turbopack/crates/turbopack-tests/tests/snapshot/intermediate-tree-shake/tree-shake-test-1/output/4c35f_tests_snapshot_intermediate-tree-shake_tree-shake-test-1_input_80df1e23._.js

```diff
@@ -0,0 +1,35 @@
+(globalThis.TURBOPACK = globalThis.TURBOPACK || []).push(["output/4c35f_tests_snapshot_intermediate-tree-shake_tree-shake-test-1_input_80df1e23._.js", {
+
+"[project]/turbopack/crates/turbopack-tests/tests/snapshot/intermediate-tree-shake/tree-shake-test-1/input/index.js [test] (ecmascript)": ((__turbopack_context__) => {
+"use strict";
+
+var { g: global, __dirname } = __turbopack_context__;
+{
+__turbopack_context__.s({});
+var __TURBOPACK__imported__module__$5b$project$5d2f$turbopack$2f$crates$2f$turbopack$2d$tests$2f$tests$2f$snapshot$2f$intermediate$2d$tree$2d$shake$2f$tree$2d$shake$2d$test$2d$1$2f$input$2f$node_modules$2f$lib$2f$index$2e$js__$5b$test$5d$__$28$ecmascript$29$__ = __turbopack_context__.i("[project]/turbopack/crates/turbopack-tests/tests/snapshot/intermediate-tree-shake/tree-shake-test-1/input/node_modules/lib/index.js [test] (ecmascript)");
+;
+console.log(`I like ${(0, __TURBOPACK__imported__module__$5b$project$5d2f$turbopack$2f$crates$2f$turbopack$2d$tests$2f$tests$2f$snapshot$2f$intermediate$2d$tree$2d$shake$2f$tree$2d$shake$2d$test$2d$1$2f$input$2f$node_modules$2f$lib$2f$index$2e$js__$5b$test$5d$__$28$ecmascript$29$__["getCat"])()}`);
+}}),
+"[project]/turbopack/crates/turbopack-tests/tests/snapshot/intermediate-tree-shake/tree-shake-test-1/input/node_modules/lib/index.js [test] (ecmascript)": ((__turbopack_context__) => {
+"use strict";
+
+var { g: global, __dirname } = __turbopack_context__;
+{
+__turbopack_context__.s({
```

Comment:
so the 'simple' part of tree shaking means the `getDog` function doesn't end up here in the context, but we still ship the actual dead function.

will deleting the 'getDog' function be part of 'intermediate' tree shaking or will that require 'advanced'?

Reply:
It will be done by the SWC minifier in a safe way. Deleting a declaration requires lots of analysis, and the minifier has one.